status/clone_project.py

```diff
@@ -26,6 +28,16 @@ class LIMSProjectCloningHandler(SafeHandler):
     """
 
     def get(self, projectid):
+        if not re.match("^(P[0-9]{3,7})", projectid):
```

Comment:
I think a comment is in place here. Try to explain that if it's not a projectid, then it's a project_name, and potentially even rename the variable.